src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
Looks like `p.sex === 'm'` is duplicated. Try to create a helper function that allows you to follow the DRY principle.
```suggestion
  const men = century
    ? people.filter((p) => isMan(p) && Math.ceil(p.died / 100) === century)
    : people.filter(isMan);
```